nav2_costmap_2d/CMakeLists.txt

```diff
@@ -32,10 +32,9 @@ nav2_package()
 
 include_directories(
   include
-  ${EIGEN3_INCLUDE_DIRS}
 )
 
-add_definitions(${EIGEN3_DEFINITIONS})
+add_definitions(${EIGEN3_DEFINITIONS}) # TODO this might not be needed...
```

Comment:
I'd like to understand what this was for, and whether it's part of the `Eigen3::Eigen` target properties. If Eigen did their cmake right, it should be.